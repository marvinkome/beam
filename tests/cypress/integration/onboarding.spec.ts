describe("Onboarding Flow", () => {
    before(() => {
        cy.visit("/app/onboarding")
    })

    it("onboards user", () => {})
})
