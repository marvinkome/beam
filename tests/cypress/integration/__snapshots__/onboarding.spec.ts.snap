exports[`Onboarding Flow > onboards user <spapshot> #0`] = `
<div class="onboarding-page">
  <header>
    <h1>Find a friend to chat with</h1>
    <p>
      <a href="/app/profile">Connect</a> your Spotify and Reddit to get more
      matches
    </p>
  </header>
  <div class="suggested-friends"></div>
  <a class="btn btn-primary" href="/app/chats">Continue to chats</a>
</div>
`;
